### ISSUE TITLE
macOS Add period with double-space

### ISSUE THREAD

**Issue by Fahrenheit17** [adobe/brackets#14000]
### Prerequisites

* [X] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [X] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [X] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

When the "Add period with double-space" setting is enabled (System Preferences - Keyboard - Text) in macOS (tested with Sierra and High Sierra), Brackets does not suppress this behavior. 

### Steps to Reproduce

1. Turn on the setting described above.
2. Type at least one alphanumeric character in Brackets and type two space characters (quickly, like double-clicking a mouse).
NOTE: Behavior appears to occur regardless of file formats; tested with PHP, Go, JavaScript, HTML, and Text, but not with every format (for practical reasons).

**Expected behavior:** This autocorrect behavior should be suppressed in code editors, as it is in Xcode.

**Actual behavior:** macOS text substitution will replace the two spaces with a space and a period.

### Versions

macOS High Sierra 10.13.2
Brackets 1.11 build 1.11.0-17524

**Comment by Svija-SVG** [comment]
I disagree… I use the macOS text substitution for several programming shortcuts, and it would be a serious handicap if they were automatically disabled. A preference would be nice, but it would add complexity.
